backend/src/main/resources/application.properties

```diff
@@ -14,3 +14,5 @@ login.defaultRedirectPath=${login_defaultRedirectPath:user}
 
 # Request API key here: https://www.themoviedb.org/documentation/api
 sefilm.tmdb.apikey=${tmdb_apikey}
+
+logging.level.didit.rocks=DEBUG
```

Comment:
semi-intentional... It did not have any effect. For some reason debug-lines for our own code isn't show, which is somewhat annoying.